raising-jiras.md

```diff
@@ -0,0 +1,87 @@
+#Raising Jiras
+
+<p>Jiras are used at Holiday Extras as a means for teams to communicate what work is required (including data and insights). This doc seeks to outline some general guidelines and best practice when writing a jira.</p>
+
+##Title:
+<p>This is a brief description of the problem to be solved. Be clear and to the point.</p>
+
+**Good example:**
+<p>"Change the endpoints in HAPI to new Horizon service for Flexible Car Hire"</p>
```

Comment:
I would prefer this to not prescribe the "how". Instead, could this be worded "Allow flexible car hire to be sold via our API"?

It's a small adjustment but it makes a difference because the solution may not be to change an endpoint thus sending the developer down a prescribed path.